css/styles.css

```diff
@@ -84,6 +84,20 @@ img {
   outline: 1px solid red;
 }
 
+.title {
+    font-family: "Bakbak One","Poppins", Arial, sans-serif;
+    font-weight: 400;
+}
+
+.header {
+    position: sticky;
+    top: 0;
+    left: 0;
+    width: 100%;
```

Comment:
You don't need to define `left` and `width` properties here - for sticky positioning `top` property is enough